app/assets/javascripts/api.jsx

```diff
@@ -0,0 +1,59 @@
+
+
+class Api {
+
+  static get(path, data = null) {
+    return Api.ajax(path, 'GET', data);
+  }
+  static post(path, data = null) {
+    return Api.ajax(path, 'POST', data);
+  }
+  static put(path, data = null) {
+    return Api.ajax(path, 'PUT', data);
+  }
+
+   static delete(path, data = null) {
+        return Api.ajax(path, 'DELETE', data);
```

Comment:
Indentation rules better when persist project-wide.
We currently using convention where indentation is 2 spaces. Check if you have our [.eslintrc](http://eslint.org/) file to copy into project root.